static/app/views/performance/transactionSummary/transactionOverview/sidebarCharts.tsx

```diff
@@ -290,7 +290,7 @@ function SidebarChartsContainer({
         axisLabel: {
           formatter: (value: number) => formatFloat(value, 1),
           color: theme.chartLabel,
-        },
+        } as any,
```

Comment:
it didn't like the formatter 🤷‍♂️

Reply:
that's because `formatFloat` returns a `number` and the type doesn't expect a number but a `string`. We have two options, we can either update the type to accept a `number` or do as following:

`formatter: (value: number) => String(formatFloat(value, 1))`